### PR TITLE
docs: add slow query docs

### PR DESCRIPTION
**Pull request by rebelice**

### CODE REVIEW

File: content/docs/slow-query/overview.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+![slow-query-sync-now](/static/docs/slow-query/slow-query-sync-now.webp)
+
+Specifically, you can view the **Slow Query Detail** for each MySQL slow query.
+
+![slow-query-detail](/static/docs/slow-query/slow-query-detail.webp)
+
+## Slow Query Weekly Summary Report
+
+Bytebase will send a weekly summary report to the workspace owner, DBA, and project owner. This feature requires configuring the SMTP server in the workspace settings. Currently, Bytebase will send the weekly summary report every Saturday at 0:00 ~ 1:00 AM.
```
Comment (tianzhou) [inline content/docs/slow-query/overview.md:53]: Link to /docs/administration/mail-delivery for the SMTP server wording

**Review by tianzhou — CHANGES_REQUESTED** [review]
Let's add a top-level section after VCS Integration

CleanShot 2023-04-26 at 13 04 59

**Comment by rebelice** [comment]
Updated. @tianzhou PTAL

**Review by tianzhou — APPROVED** [review]
LGTM. Good

**Pull request merged**